docs/contracts/v3/reference/deployments/Blast-Deployments.md

```diff
@@ -0,0 +1,55 @@
+---
+id: blast-deployments
+title: Blast Deployments
+---
+
+# Uniswap Contract Deployments
+
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and `@uniswap/swap-router-contracts` are deployed at the addresses listed below. Integrators should **no longer assume that they are deployed to the same addresses across chains** and be extremely careful to confirm mappings below.
+
+| Contract                                                                                                                                                     | Blast addresses                              |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0x792edAdE80af5fC680d96a2eD80A44247D2Cf6Fd` |
+| [Multicall](https://blastscan.io/address/0xdc7f370de7631ce9e2c2e1dcda6b3b5744cf4705#code)                                                                    | `0xdC7f370de7631cE9e2c2e1DCDA6B3B5744Cf4705` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0x7C9cAa4ac84C8FAD8Bd504DBF90e791F91f41705` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0x2E95185bCdD928a3e984B7e2D6560Ab1b17d7274` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0xAa32bD3926097fd04d22b4433e9867417EE79333` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x497089D9450BB58f536c38c1C0d0A37472303508` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0xB22Ef02E13B1900EBF10391e57162402c11BfF05` |
```

Comment:
also not verified, worth noting so we can try to

Reply:
Got the names from the Blast deployment team in slack, and verified contracts on the deployer address